src/addiction.cpp

```diff
@@ -116,7 +116,7 @@ void addict_effect( player &u, addiction &add )
             if( one_in( 20 ) && dice( 2, 20 ) < in ) {
                 u.add_msg_if_player( m_bad, _( "Your hands start shaking... you need some painkillers." ) );
                 u.add_morale( MORALE_CRAVING_OPIATE, -40, -10 * in );
-                u.add_effect( effect_shakes, 2_minutes + in * 5_turns );
+                u.add_effect( effect_shakes, 2_minutes + in * 30_turns );
```

Comment:
30_seconds or 5_seconds?

Reply:
30_seconds. Missed that when I was changing them.